src/main/java/dinkplugin/util/Utils.java

```diff
@@ -190,10 +190,14 @@ public AccountType getAccountType(@NotNull Client client) {
     }
 
     @Nullable
-    public String getChatBadge(@NotNull AccountType type, boolean seasonal) {
+    public String getChatBadge(@NotNull AccountType type, boolean seasonal, DinkPluginConfig config) {
         if (seasonal) {
```

Comment:
are we fine with seasonal badge taking precedence over custom badge?

Reply:
I think allowing custom badge over seasonal badge would be the better approach